core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -71,6 +71,7 @@
 
     private Duration statusRefreshMaxWait = new Duration(1, TimeUnit.SECONDS);
     private Duration infoUpdateInterval = new Duration(3, TimeUnit.SECONDS);
+    private Duration taskTerminationTimeout = new Duration(10, TimeUnit.MINUTES);
```

Comment:
Yeah, I didn't think too hard about what the right value should be because I wasn't sure how long we would expect this to take. Having tested this a bit, termination is pretty typically in the 10-20ms range so 1 minute sounds like a better default to me- I'll change it in the next revision.